### DISCUSSION TITLE
UI bug when changing number of indexers per page

### DISCUSSION THREAD

**Discussion by jim-counter** [subquery/subql#932] [Frontier TestNet - Bug Reporting]
Changing the number of indexers to display per page does not seem to be working. Present on https://frontier.subquery.network/staking/indexers

![image](https://user-images.githubusercontent.com/11335994/165607464-acfa160b-e91a-483c-97f4-93066c48f9ce.png)

**Comment by mzxyz** [comment]
@CaiYiLiang

**Comment by lilian79511** [comment]
Ticket created: SQN-417
